app/src/main/res/layout/fragment_feedback.xml

```diff
@@ -101,8 +102,9 @@
                 android:layout_height="wrap_content"
                 android:background="#00ffffff"
                 android:focusable="false"
-                android:paddingTop="10dp"
-                android:text="May 2010" />
+                android:hint="May 2010"
```

Comment:
I don't think that should be a hint, should it? In other layouts we use `tools:text=""` for text that is for testing purposes only.

Reply:
In this case, it should not be hard coded then. And it should be something more helpful; we should make the label TextView a TextInputLayout so that the hint goes atop the EditText once something is put in.

Reply:
I think the original value was just there to have a placeholder to see something in the designer. I agree, that we might want a hint, though.

Reply:
On second thought, this text is changed to be the current month+year when the fragment is created. It has a label "When we met" and when clicking it a date picker shows up. Does it still need a hint?

Reply:
Yes it does—to show that the value can be changed. Especially with the material design language, it often isn’t immediately visible whether something can be clicked on or not.